### PR TITLE
fix(notif): fixed typo in pushover hint

### PR DESCRIPTION
**Pull request by danshilm**
#### Description
Simple fix for a typo

#### Screenshot (if UI-related)
None

#### To-Dos
None

- [x] Successful build `yarn build`
- [x] Ran `yarn i18n:extract`

#### Issues Fixed or Closed
- Fixes none

### CODE REVIEW

File: src/components/Settings/Notifications/NotificationsPushover/index.tsx [hunk 1]
```diff
@@ -24,7 +24,7 @@ const messages = defineMessages({
   test: 'Test',
   settinguppushover: 'Setting Up Pushover Notifications',
   settinguppushoverDescription:
-    'To configure Pushover notifications, you will need to <RegisterApplicationLink>register an application</RegisterApplicationLink> and enter the API key below.\
+    'To configure Pushover notifications, you will need to <RegisterApplicationLink>register an application</RegisterApplicationLink> and enter the API token below.\
     (You can use one of our <IconLink>official icons on GitHub</IconLink>.)\
     You will need also need your user key.',
```
Comment (TheCatLady) [inline src/components/Settings/Notifications/NotificationsPushover/index.tsx:29]:
```suggestion
    You will also need your user key.',
```

**Comment by TheCatLady** [comment]
Also need to update the strings in `en.json`. 😉  Thanks for submitting the PR, @danshilm!

**Comment by danshilm** [comment]
Hmm, `i18n:extract` didn't update the strings in the locale files, is that normal?
Made the changes in the locale files anyway.

**Comment by TheCatLady** [comment]
> Hmm, `i18n:extract` didn't update the strings in the locale files, is that normal?
> Made the changes in the locale files anyway.

`yarn i18n:extract` will not overwrite existing strings, but only add/remove strings.

**Review by TheCatLady — APPROVED** [review]

**Review by sct — APPROVED** [review]

**Pull request merged**